pkg/gatewayserver/observability.go

```diff
@@ -334,6 +334,6 @@ func registerSuccessDownlink(ctx context.Context, gtw *ttnpb.Gateway, protocol s
 }
 
 func registerFailDownlink(ctx context.Context, gtw *ttnpb.Gateway, ack *ttnpb.TxAcknowledgment, protocol string) {
-	events.Publish(evtTxFailureDown.NewWithIdentifiersAndData(ctx, gtw, ack.Result))
```

Comment:
Can we add `result` as a label to `downlinkTxFailed` and record the result please ?

Reply:
Yeah I thought of that but then we add potentially 8 new series of data and we've never really used this for debugging. Doesn't hurt I guess.

Reply:
It would be nice for https://github.com/TheThingsNetwork/lorawan-stack/issues/18